task/zap-baseline/0.1/README.md

```diff
@@ -0,0 +1,31 @@
+# Zap proxy
+
+[Zap proxy](https://github.com/zaproxy/zaproxy) baseline is a command line tool for security scanning endpoints.
+
+## Install the Task
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/zap-baseline/0.1/zap-baseline.yaml
+```
+
+## Parameters
+
+- **testURL**: The URL to be scanned.
+- **configFile**: Path to zap-scan-config, is stored under /zap/wrk/, ca you only set the file name.
```

Comment:
```suggestion
- **configFile**: Path to zap-scan-config, is stored under /zap/wrk/, you only set the file name.
```
I'm not sure what this should be changed to "ca you only set the file name" - it may be missing something -  or I may be missing something or the context.